### PR TITLE
Added dimensions to package, and completed the conversations model

### PR DESCRIPTION
**Pull request by me4502**
Fixes the following:
#49

### CODE REVIEW

**Comment by me4502** [comment]
@KnightHawk3 review pls

File: packr/models.py [hunk 1]
```diff
@@ -169,3 +171,38 @@ def __init__(self, name, **kwargs):
...
+
+    messages = relationship('Message', backref='conversation')
+
+    def __init__(self, name, **kwargs):
+        """Create instance."""
+        db.Model.__init__(self, name=name, **kwargs)
+
+    def __repr__(self):
+        """Represent instance as a unique string."""
+        return '<Message({id})>'.format(id=self.id)
```
Comment (ZeroEcks) [inline packr/models.py]: This should say something else instead of Message

**Review by ZeroEcks — CHANGES_REQUESTED** [review]
Just fix these things and we are good to go

File: packr/models.py (hunk 1 above)
Comment (me4502) [reply packr/models.py]: No it shouldn't

Comment (ZeroEcks) [reply packr/models.py]: fite me irl

**Review by ZeroEcks — APPROVED** [review]
ty

**Pull request merged**